drivers/wifi/nrf700x/osal/fw_if/umac_if/src/fmac_api.c

```diff
@@ -3329,4 +3329,46 @@ enum wifi_nrf_status wifi_nrf_fmac_twt_teardown(void *dev_ctx,
 
 	return status;
 }
+
+enum wifi_nrf_status wifi_nrf_fmac_set_multicast(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
+				 unsigned char if_idx,
+				 struct nrf_wifi_umac_mcast_cfg *mcast_info)
+{
+	enum wifi_nrf_status status = WIFI_NRF_STATUS_FAIL;
+	struct nrf_wifi_umac_cmd_mcast_filter *set_mcast_cmd = NULL;
+
+	unsigned int len = sizeof(*set_mcast_cmd);
```

Comment:
Why is extra variable len required...why not just use sizeof(*set_mcast_cmd) where needed?